packages/react-ui/src/Grid/index.js

```diff
@@ -17,15 +18,15 @@ export const Grid = styled.ul`
 `;
 
 export const GridCell = styled.li`
-  flex-basis: calc(100% / 4 - 2 * ${spacing.small});
+  display: flex;
   flex-grow: 0;
   flex-shrink: 1;
+  width: calc(100% / 4 - 2 * ${spacing.small});
```

Comment:
it's not clear what was wrong with flex-basis that width solves  ?

Reply:
It solves nor changes absolutely nothing, I was tinkering with IE11 and forgot to put the old value back !